### PR TITLE
Add Helm Chart

### LINKED ISSUE #12: Add Helm charts
Add Helm charts for each of current projects.

### PR DESCRIPTION
**Pull request by Wizofgoz**
@StephenSorriaux Here's a first crack at a chart that's able to use any version. Let me know if you have any concerns!

closes #12

### CODE REVIEW

File: charts/mangos/templates/mysql-statefulset.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+{{ toYaml . | indent 8 }}
+    {{- end }}
+  volumeClaimTemplates:
+  - metadata:
+      name: data
+    spec:
+      accessModes: ["ReadWriteOnce"]
+      resources:
+        requests:
+          storage: 10Gi
```
Comment (StephenSorriaux) [inline charts/mangos/templates/mysql-statefulset.yaml]: Can the storage be defined dynamically?

File: charts/mangos/templates/secret.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{{- define "secret.login" }}
+    {{ template "chart.fullname" . }}-mysql-service;3306;root;{{ template "getPassword" . }};realmd
+{{- end }}
+
+{{- define "secret.world" }}
+    {{ template "chart.fullname" . }}-mysql-service;3306;root;{{ template "getPassword" . }};mangos{{ .Values.wowVersion }}
```
Comment (StephenSorriaux) [inline charts/mangos/templates/secret.yaml]: For mangos 2 & 3, databases are "mangos" and "characters"

**Review by StephenSorriaux — CHANGES_REQUESTED** [review]
Very nice PR, is it possible to add some instructions in the README.md file?

File: charts/mangos/templates/secret.yaml (hunk 2 above)
Comment (Wizofgoz) [reply charts/mangos/templates/secret.yaml]: You're right, I updated it to address this

File: charts/mangos/templates/mysql-statefulset.yaml (hunk 1 above)
Comment (Wizofgoz) [reply charts/mangos/templates/mysql-statefulset.yaml]: Good idea, I just added this

**Comment by Wizofgoz** [comment]
@StephenSorriaux just added to the readme and realized I didn't have a way to show the DB password so I added it to the `NOTES.txt` file

**Review by StephenSorriaux — APPROVED** [review]
Looks great, thanks a lot!

**Pull request merged**